docs/triage.md

```diff
@@ -0,0 +1,62 @@
+## Introduction
+
+This document will outline the process of triaging GitHub issues and provide an explanation of GitHub labels.
+
+## GitHub Labels
+
+Labels that starts with `a:` are area labels.
+Area labels help categorize issues based on their scope. Each issue should ideally have at least one area label.
+The description of the area label ends with a code that indicates the maintainer or backup maintainer of that area.
+For example, if a label description ends with `(mX,bY)`, it means person X is the maintainer, and person Y is the backup maintainer for that area.
+
+Apart from area labels, there can be other labels used to provide additional context to issues.
+
+- **Bug**: Indicates that the issue is a bug or a software defect.
+- **Enhancement**: Indicates that the issue suggests an improvement or new feature.
+- **Good First Issue**: Indicates that the issue is suitable for newcomers to contribute to the project.
+   The issue should have a good description and ideally a comment to describe how to do it
+- **Needs More Info**: Indicates that the issue lacks sufficient information and requires additional details from the reporter.
+  The maintainer for the area must remove the tag when more info is provided, or close it if too long
+
+## GitHub Assignee
+
+Here are some guidelines for GitHub assignees:
+- Assign issues to individuals who are actively working on or responsible for a particular area.
```

Comment:
I'd like to suggest this edit to basically avoid interpreting this as auto-area-assignment.

```suggestion
- Assign issues to individuals who are actively working on them or going to actively work on them in the near future.
```

That said, in the light of the bullet points right below, I think this one could also be omitted. The ones following this one are very clear and instructive (and good).